test/rust_src/src/windows-tests.el

```diff
@@ -105,3 +105,34 @@
   (should (= 1.0 (window-normal-size w1 't)))
   (should (= 0.5 (window-normal-size w2)))
   (should (= 1.0 (window-normal-size w2 't)))))
+
+(ert-deftest window-pixel-top ()
+  (let ((w1 (selected-window))
+        (w2 (split-window-vertically)))
+    (should (= (window-pixel-top) (window-pixel-top w1)))
+    (should-not (= (window-pixel-top w1) (window-pixel-top w2)))))
+
+(ert-deftest window-fringes ()
+  (let ((w1 (selected-window))
+        (w2 (split-window)))
+    (set-window-fringes w1 4 4 nil)
+    (set-window-fringes w2 16 16 t)
+    (should (equal `(4 4 nil) (window-fringes w1)))
+    (should (equal `(4 4 nil) (window-fringes nil)))
+    (should (equal `(16 16 t) (window-fringes w2)))))
+
```

Comment:
None of these check the return value of `set-window-fringes`. Can there be a test which validates the nil return?

Reply:
Sure, forgot about this. I have added this code now in:
 - https://github.com/remacs/remacs/pull/1367/commits/d265174a2fac343a81e3b1a7e8bc671bd8658534